### PR TITLE
SUBMARINE-213. [SDK] Add pysubmarine-tracking example in mini-submarine

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?
Add a tracking api example in mini-submarine

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-213

### How should this be tested?
https://travis-ci.org/pingsutw/hadoop-submarine/builds/599649334

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-sdk/pysubmarine/submarine/store/database/models.py [hunk 1]
```diff
@@ -37,17 +37,17 @@
 class SqlMetric(Base):
     __tablename__ = 'metrics'
 
-    key = Column(String(250))
+    key = Column(String(190))
```
Comment (liuxunorg) [inline submarine-sdk/pysubmarine/submarine/store/database/models.py]: Why need change String size?

File: docs/submarine-sdk/pysubmarine/tracking.md [hunk 2]
```diff
@@ -20,6 +20,16 @@
 - [TODO] (WEB) Metric tracking ui in workbench-web
 - [TODO] (WEB) Metric graphical display in workbench-web
 
+## Quickstart
+1. Start mini-submarine
+
```
Comment (liuxunorg) [inline docs/submarine-sdk/pysubmarine/tracking.md]: Can you add a `start mini-submarine` command here?

File: docs/submarine-sdk/pysubmarine/tracking.md [hunk 3]
```diff
@@ -20,6 +20,16 @@
...
+2. Start Mysql server in mini-submarine
```
Comment (liuxunorg) [inline docs/submarine-sdk/pysubmarine/tracking.md]: Can you add a `start mysql` command here?

File: docs/submarine-sdk/pysubmarine/tracking.md (hunk 2 above)
Comment (pingsutw) [reply docs/submarine-sdk/pysubmarine/tracking.md]: I add a link to [mini-submarine/README.md](https://github.com/pingsutw/hadoop-submarine/tree/SUBMARINE-213/dev-support/mini-submarine#run-mini-submarine-image), so users can get more information about how to start and build mini-submarine

**Comment by liuxunorg** [comment]
LGTM.

**Comment by pingsutw** [comment]
Thanks @liuxunorg for the review

**Pull request closed without merging**